plugin/storage/es/dependencystore/schema.go

```diff
@@ -22,3 +22,17 @@ const dependenciesMapping = `{
       "` + dependencyType + `":{}
    }
 }`
+
+const dependenciesMapping7 = `{
```

Comment:
Do we need this? The legacy mapping should work with the ES7. I think you added this after I tested this and it worked without this change. 

If it works please revert this.

Reply:
The legacy mapping does not work with ES7 because the legacy mapping includes the type name.  Type names are not permitted in ES7 unless you use the include_type_name parameter, which we removed because it is deprecated.